templates/admin/user/index.html.twig

```diff
@@ -50,6 +52,12 @@
                             <td>{{ user.id }}</td>
                             <td>{{ user.username }}</td>
                             <td>{{ user.email }}</td>
+                            <td>{% if user.activated == 1 %}
```

Comment:
not sure, but I think you can use `{{ user.activated ? 'true' : 'false' }}`

Reply:
{{ user.activated ? 'true' : 'false' }} Worked perfectly